packages/react-components/src/components/BrowserPermissionDeniedIOS.tsx

```diff
@@ -34,7 +34,7 @@ export interface BrowserPermissionDeniedIOSProps extends BrowserPermissionDenied
   /**
    * Localization strings for BrowserPermissionDeniedIOS component.
    */
-  strings: BrowserPermissionDeniedIOSStrings;
+  strings?: BrowserPermissionDeniedIOSStrings;
```

Comment:
If these strings are now optional we should make sure that the component is getting its own localized strings instead of its parent.

Reply:
They are using localized strings

Reply:
const locale = useLocale().strings.BrowserPermissionDeniedIOS;

Reply:
Oh I didnt see it in the expanded file, will look again 😊